src/extensions/vb/Microsoft.DotNet.UpgradeAssistant.Extensions.VisualBasic/MyTypeTargetFrameworkSelectorFilter.cs

```diff
@@ -37,9 +37,9 @@ public void Process(ITargetFrameworkSelectorFilterState tfm)
 
             if (_windowsMyTypes.Contains(myType))
             {
-                _logger.LogInformation("Project {Name} contains MyType node that requires at least net5.0-windows.", tfm.Project);
+                _logger.LogInformation("Project {Name} contains MyType node that requires at least {framework}-windows.", tfm.Project, tfm.AppBase);
```

Comment:
```suggestion
                var final = tfm.AppBase with { Platform = TargetFrameworkSelector.Platforms.Windows };
                _logger.LogInformation("Project {Name} contains MyType node that requires at least {framework}", tfm.Project, final);
                
                tfm.TryUpdate(final);
```

I believe this requires at windows as well. You can update the app base to have that platform (not sure if the code is right, but there are constants somewhere to do this pattern).

Reply:
Thanks, I'll run w/ this and see what happens but also check if I can find the constants; though I am not sure they'd work because `.AppBase` can differ based on the user's choice of TFM from the cmd-line, right?

Reply:
AppBase should be a parsed tfm